python/lsst/daf/butler/cli/cmd/commands.py

```diff
@@ -50,7 +49,7 @@ def butler_import(*args, **kwargs):
 
 @click.command()
 @repo_argument(help=repo_argument.will_create_repo)
-@config_file_option(help="Path to an existing YAML config file to apply (on top of defaults).")
+@click.option("--seed-config", help="Path to an existing YAML config file to apply (on top of defaults).")
 @click.option("--standalone", is_flag=True, help="Include all defaults in the config file in the repo, "
               "insulating the repo from changes in package defaults.")
 @click.option("--override", "-o", is_flag=True, help="Allow values in the supplied config to override any "
```

Comment:
We probably want to remove the `-o` here since it's a very rare option and it's likely that we'll be using `-o` elsewhere.

Reply:
Also change text to "...supplied seed config to override all repo defaults"